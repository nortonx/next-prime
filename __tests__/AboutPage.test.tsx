import { render, screen } from "@testing-library/react"
import AboutPage from "@/app/about/page"

describe("About Page", () => {
  it("should render and match snapshot", async () => {
    const { container } = render(<AboutPage/>)
    const page = await screen.findByTestId("about-page")
    expect(page).toBeInTheDocument()
    expect(container).toMatchSnapshot()
  })
})