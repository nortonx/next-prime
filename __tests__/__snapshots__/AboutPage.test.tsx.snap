// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Page should render and match snapshot 1`] = `
<div>
  <main
    class="flex align-items-center justify-content-center"
    data-testid="about-page"
  >
    <h1>
      About Page
    </h1>
    <div
      class="form-row"
    >
      <label
        for="city"
      >
        City:
      </label>
      <input
        class="p-inputtext p-component"
        data-pc-name="inputtext"
        data-pc-section="root"
        id="city"
        list="cities"
        name="city"
      />
      <datalist
        id="cities"
      >
        <option
          value="bh"
        />
        <option
          value="sp"
        />
        <option
          value="rj"
        />
      </datalist>
    </div>
  </main>
</div>
`;
